structuring-content/activity-personal-portfolio-1.md

```diff
@@ -0,0 +1,60 @@
+# Activity: Structuring Personal Portfolio
+
+## Goals
+
+Our coding skills improve with repetition.
+
+In this first activity, we will build the HTML structure of our personal portfolio website using semantic HTML tags, as well as proper indentation to show the relationship between parent and child elements.
+
+## Activity Instructions
+
+### Setup
+
+Fork the following repo: [personal portfolio website](), then clone it to your local machine. From here you can begin creating the following pages based on the requirements below.
+
+### Requirements
+
+This portfolio should have the following pages:
+- `index.html`
+- `about.html`
+- `portfolio.html`
```

Comment:
I suggest we use something like what we did in the JS sections.  Although it won't let me insert 3 backtics around it to make it look better. 

```suggestion
.
├── index.html
├── about.html
└── portfolio.html
```